packages/frontend/src/components/MkChart.vue

```diff
@@ -106,7 +114,8 @@ const getColor = (i) => {
 	return colorSets[i % colorSets.length];
 };
 
-const now = new Date();
+// eslint-disable-next-line vue/no-setup-props-destructure
+const now = props.nowForChromatic != null ? new Date(props.nowForChromatic) : new Date();
```

Comment:
型定義から見ると`props.nowForChromatic`の比較はundefinedとの厳密等価での比較でも問題なさそうに見えますがnullとの比較にしている理由はなにかありますか？
```suggestion
const now = props.nowForChromatic !== undefined ? new Date(props.nowForChromatic) : new Date();
```

Reply:
レビューありがとうございます。これは以前しゅいろさんに頂いたレビュー https://github.com/misskey-dev/misskey/pull/12923#discussion_r1443875841 に基づいて書いています（ここで undefined を明示的に区別する必要はないため）

Reply:
なるほど（ならOKです👌次からレビュー時には気にしないことにします）

Reply:
（最初、自分も気になっていたのですが、[ここ](https://typescript-jp.gitbook.io/deep-dive/recap/null-undefined#dochiradearukawochekkusuru)をみて勝手に納得していました）